lib/debase.rb

```diff
@@ -1,5 +1,10 @@
 if defined?(RUBY_ENGINE) && 'rbx' == RUBY_ENGINE
   require 'debase/rbx'
+elsif defined?(RUBY_ENGINE) && RUBY_ENGINE == 'truffleruby'
+  require "debase/version"
+
+  Debugger = Module.new
```

Comment:
The reason for this is explained in the commit message: https://github.com/ruby-debug/debase/pull/106/commits/06bdb3c03fcf023f5256c06cb7fa97043c9f9c5a

But maybe it's the wrong place to do this and instead we should change lib/ruby-debug-ide.rb to noop earlier.